ui/app/templates/components/namespace-picker.hbs

```diff
@@ -32,6 +32,15 @@
         </div>
       </div>
       <div class="namespace-list {{if this.isAnimating 'animated-list'}}">
+        {{#if this.auth.isRootToken}}
```

Comment:
I opted to follow what's being done in the user menu dropdown see [here](https://github.com/hashicorp/vault/blob/e5432b0577f162b90c5b818989a91fe1b4c2cafc/ui/app/components/sidebar/user-menu.hbs#L27). hen we use `AlertInline`, with type `warning` the icon looks really small and styling gets a little wonky and with type `compact` the namespace picker dropdown gets pulled to the top (see attached images).